app/src/main/java/com/keylesspalace/tusky/db/TimelineStatusEntity.kt

```diff
@@ -50,15 +65,19 @@ data class TimelineStatusEntity(
     val bookmarked: Boolean,
     val favourited: Boolean,
     val sensitive: Boolean,
-    val spoilerText: String?,
-    val visibility: Status.Visibility?,
+    val spoilerText: String,
+    val visibility: Status.Visibility,
     val attachments: String?,
     val mentions: String?,
     val application: String?,
     val reblogServerId: String?, // if it has a reblogged status, it's id is stored here
     val reblogAccountId: String?,
     val poll: String?,
-    val muted: Boolean?
+    val muted: Boolean?,
+    val expanded: Boolean, // used as the "loading" attribute when this TimelineStatusEntity is a placeholder
```

Comment:
Could we do this nicer? Is it worth having an additional column "loading" in the db?

Reply:
We could have it as number to map it to enum but it seems like none of these needs to be in DB anyway